src/Native/System.Native/pal_networking.h

```diff
@@ -132,5 +139,7 @@ extern "C" Error SetIPv4Address(uint8_t* socketAddress, int32_t socketAddressLen
 extern "C" Error GetIPv6Address(
     const uint8_t* socketAddress, int32_t socketAddressLen, uint8_t* address, int32_t addressLen, uint32_t* scopeId);
 
-extern "C" Error SetIPv6Address(
-    uint8_t* socketAddress, int32_t socketAddressLen, uint8_t* address, int32_t addressLen, uint32_t scopeId);
+extern "C" int32_t GetControlMessageBufferSize(int32_t isIPv4, int32_t isIPv6);
```

Comment:
NIT: please add the comment header to all new functions describing what the function does (briefly) and what it returns on success / failure